frontend/env.ts

```diff
@@ -5,7 +5,7 @@ export const env = createEnv({
   clientPrefix: 'VITE_',
 
   client: {
-    VITE_API_URL: z.url(), // no trailing slash
+    VITE_API_URL: z.string(), // no trailing slash
```

Comment:
locally i have .env in the project root containing django env vars and a .env in frontend containing vite env vars. Not attached to this setup, but seems to make sense to have them separate. 

Possibly relevant info: frontend env vars will be baked in at build time and don't need to be env vars in the image we run.

Reply:
The main reason I noted it in the comment is it looked like according to how the .env examples are set up, we had JIRA_* set up in the frontend, which was confusing me a bit

Reply:
FWIW I think this is fine for now, its mostly a dev detail. I still need to configure these secrets in Cloud Run anyway